lms/services/roster.py

```diff
@@ -319,12 +319,28 @@ def fetch_canvas_group_roster(self, canvas_group: LMSSegment) -> None:
         )
         application_instance = self._get_application_instance(lms_course)
 
-        roster = self._lti_names_roles_service.get_context_memberships(
-            application_instance.lti_registration,
-            # We won't use the names and roles endpoint for groups, we need to pass a URL from the Canvas extension to the API.
-            # https://canvas.instructure.com/doc/api/names_and_role.html#method.lti/ims/names_and_roles.group_index
-            f"https://{application_instance.lms_host()}/api/lti/groups/{canvas_group.lms_id}/names_and_roles",
-        )
+        try:
+            roster = self._lti_names_roles_service.get_context_memberships(
+                application_instance.lti_registration,
+                # We won't use the names and roles endpoint for groups, we need to pass a URL from the Canvas extension to the API.
+                # https://canvas.instructure.com/doc/api/names_and_role.html#method.lti/ims/names_and_roles.group_index
+                f"https://{application_instance.lms_host()}/api/lti/groups/{canvas_group.lms_id}/names_and_roles",
+            )
+
+        except ExternalRequestError as err:
+            ignored_errors = [
```

Comment:
This looks like overkill as we only have one error type now but we have the same code already in this module.

If this keeps growing we could use a helper method to abstract it.